test/Microsoft.DotNet.Docker.Tests/ImageTests.cs

```
@@ -99,7 +99,12 @@ private static string GetFilterRegexPattern(string filterEnvName)
                 if (!imageData.HasNoSdk)
                 {
                     VerifySdkImage_PackageCache(imageData);
-                    await VerifySdkImage_RunApp(imageData, appSdkImage);
+
+                    // TODO: Skip running app in arm + web configuration to workaround https://github.com/dotnet/cli/issues/9162
+                    if (!(imageData.IsArm && imageData.IsWeb))
```

https://github.com/dotnet/cli/pull/9163 has been merged so dotnet/cli#9162 should be fixed going forward. Instead of skipping the ARM web tests in code is there no way to mark it as a known failure?

---

dotnet/cli#9162 is fixed but not in RC.  Because we are using [Theory] it is not practical to mark the aspnet arm scenarios as known failures.  Additionally, this causes the test scenario to fail immediately, while this workaround allows the rest of the scenario to run - e.g. testing the runtime images.